backend/api/tests/test_admin_views.py

```
@@ -406,4 +406,130 @@ def test_promote_unauthorized(self, user_client, regular_user):
+
+        assert response.status_code == status.HTTP_200_OK
+        assert len(response.data) == 1
+        assert response.data[0]['filename'] == 'test_recording.webm'
+
+        # Filter by new video
+        response = admin_client.get(url, {'video_id': new_video.id})
+        assert response.status_code == status.HTTP_200_OK
+        assert len(response.data) == 1
+        assert response.data[0]['filename'] == 'another_recording.webm'
+
+    def test_filter_by_status(self, admin_client, test_webcam_recording):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_filter_by_status`

Add return type annotation: `None`

(ANN201)